kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -645,6 +774,20 @@ define([
                 );
             }
             this.tour.start();
+        },
+
+        uuidv4: function (a, b) {
+            for (
```

Comment:
Is it worth install the `npm uuid` package to avoid making our own?

Reply:
I think it is installed somewhere in the repository but I couldn't get it working (I do see it under node_modules, but including it via require didn't work). Everywhere else I searched through the repo for uuid they were using this same method. I'm open though - just got too lazy to troubleshoot further

Reply:
That's fine! I just was curious about it. That sounds like it should be a backlog item to address.